akka-sample-camel-java/README.md

```diff
@@ -1,12 +1,12 @@
-This tutorial contains 3 samples of [Akka Camel](http://doc.akka.io/docs/akka/2.5.0/java/camel.html).
+This tutorial contains 3 samples of [Akka Camel](http://doc.akka.io/docs/akka/current/java/camel.html).
```

Comment:
Oh I just noticed we have a '2.5' branch here - might need some more thought then :)